packages/lisk-api-client/package.json

```diff
@@ -41,16 +41,20 @@
 		"test:watch:min": "npm run test:watch -- --reporter=min",
 		"test:node":
 			"npm run build:check && BABEL_ENV=buildcheck mocha test --compilers js:babel-register --recursive",
-		"test:browser": "cypress run --env ROOT_DIR=\"${PWD##*/}\"",
-		"serve:browsertest": "http-server browsertest",
+		"pretest:browser":
+			"npm run build:browsertest && http-server -p 8080 browsertest &",
```

Comment:
Looking at this maybe you're right that `build:browsertest` should include `npm run build:node`.